src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -43,6 +47,15 @@ var _ webhook.Defaulter = &Cluster{}
 // TODO(user): fill in your defaulting logic.
 func (r *Cluster) Default() {
 	log.Info("default", "name", r.Name)
+	if *r.Spec.Replicas >= 3 {
```

Comment:
nice, let's please document this somewhere in types, maybe next to replicas field?

Reply:
I addressed it please verify.